src/api/coderApi.ts

```diff
@@ -33,17 +33,12 @@ import { createHttpAgent } from "./utils";
 
 const coderSessionTokenHeader = "Coder-Session-Token";
 
-type WorkspaceConfigurationProvider = () => WorkspaceConfiguration;
```

Comment:
Was removing the provider related to the disposing in any way?  Or just kinda a "while I am here" sort of thing?